### ISSUE TITLE
DVN 4.0 Install Demo ?

### ISSUE THREAD

**Issue by din2011** [dvn/dvn-install-demo#4]
Can anyone provide virtual box / vagrant for Dataverse 4.0 ? (latest version)

**Comment by pdurbin** [comment]
Yes! There is a Vagrant file at the root of the Dataverse git repo: https://github.com/IQSS/dataverse

Here are some docs about it: http://guides.dataverse.org/en/4.0/developers/tools.html#vagrant
